apps/explorer/priv/repo/migrations/20220426113555_create_smart_contracts_transaction_counts_view.exs

```diff
@@ -0,0 +1,17 @@
+defmodule Explorer.Repo.Migrations.CreateSmartContractsTransactionCountsView do
+  use Ecto.Migration
+
+  def up do
+    execute("""
+    create materialized view if not exists smart_contract_transaction_counts as
```

Comment:
I'd suggest to stick to the standard of using uppercase for SQL queries as it makes it more readable and allows scanning relation names quickly, some formatting would also be nice.

```
CREATE MATERIALIZED VIEW IF NOT EXISTS smart_contract_transaction_counts AS
WITH last_block_number AS (
  SELECT
    MAX(number) - 17280 * 90 AS number
  FROM blocks
)
SELECT
  to_address_hash AS address_hash,
  COUNT(*) AS transaction_count
FROM transactions
WHERE
  to_address_hash IN (SELECT address_hash FROM smart_contracts) AND
  block_number > (SELECT number FROM last_block_number)
GROUP BY to_address_hash;
```